server/src/routines/matchmaking.ts

```diff
@@ -47,11 +47,11 @@ function setupGame(
 	let game = new GameModel(
 		{
 			model: player1,
-			deck: player1.deck.cards.map((card) => card.props.numericId),
+			deck: player1.deck!.cards.map((card) => card.props.numericId),
 		},
 		{
 			model: player2,
-			deck: player2.deck.cards.map((card) => card.props.numericId),
+			deck: player2.deck!.cards.map((card) => card.props.numericId),
```

Comment:
I dont think using the bang operator here is a very good idea

Reply:
Fixed this